src/js/components/FileInput/stories/Multiple.js

```diff
@@ -8,7 +8,9 @@ export const Multiple = () => (
     <Box fill align="center" justify="start" pad="large">
       <Box width="medium">
         <FileInput
-          multiple
+          multiple={{
```

Comment:
Getting an error in the "Multiple" story:
<img width="1015" alt="Screen Shot 2021-07-28 at 9 17 03 AM" src="https://user-images.githubusercontent.com/12522275/127359091-9c2d6d49-7998-45da-8536-f41301b93180.png">

Reply:
I resolve this one.